mdv/markdownviewer.py

```diff
@@ -130,7 +130,7 @@
 
 import sys
 # grrr
-PY3 = True if sys.version_info[0] > 2 else False
+PY3 = sys.version_info[0] > 2
```

Comment:
wouldn't `sys.version_info.major` be more explicit?

Reply:
Good point!